### DISCUSSION TITLE
Checkpointing with Loom

### DISCUSSION THREAD

**Discussion by matanmarkind** [tokio-rs/tokio#4156] [Q&A] [answered]
I am trying to debug a loom test which is failing, but I am having issues using the checkpoint.

When I run:
$ RUST_BACKTRACE=full RUSTFLAGS='--cfg loom' cargo +nightly test --test loom --release

I get: "  process didn't exit successfully: . . . (signal: 4, SIGILL: illegal instruction)"

When I try to run with checkpoint:
$ LOOM_CHECKPOINT_FILE=fpath.txt RUST_BACKTRACE=full RUSTFLAGS='--cfg loom --cfg feature="checkpoint"' cargo +nightly test --test loom --release

I get a failure to compile due to: "unresolved import `serde`"

Am I configuring my compilation incorrectly?

**Comment by Darksonn** [comment, marked as answer]
Is this a question about use of loom in Tokio, or use of loom for some other project? Also, please include full compiler errors.

**Comment by matanmarkind** [comment]
Ah sorry about that, fixed and opened an issue in the loom crate.

https://github.com/tokio-rs/loom/issues/233
